fixtures/blizzard/lil-g.json

```
@@ -0,0 +1,390 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Lil' G",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ryan Goodwin"],
+    "createDate": "2023-05-24",
+    "lastModifyDate": "2023-05-24"
+  },
+  "links": {
+    "manual": [
+      "https://cdn.shopify.com/s/files/1/1741/5055/files/lil-g_manual_rev_a.pdf"
+    ],
+    "productPage": [
+      "https://www.blizzardpro.com/products/lil-g"
+    ],
+    "video": [
+      "https://www.facebook.com/interstatemusic/videos/blizzard-lighting-lil-g-moving-head/502760530834982/"
```

Facebook videos are hard to embed, so let's better use the same video on YouTube:

```suggestion
      "https://www.youtube.com/watch?v=-lwu9weWlBk"
```

---

I'm glad you were able to find that video because I could not seem to find one when I looked.